app/src/main/res/layout/context_menu.xml

```diff
@@ -12,16 +11,20 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_margin="0dp"
-        android:padding="0dp"
-        android:background="@android:color/transparent" />
+        android:background="@android:color/transparent"
+        android:paddingTop="0dp"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        android:paddingBottom="16dp" />
 
     <TextView
         android:id="@+id/warning"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:background="#ff262626"
+        android:paddingHorizontal="24dp"
+        android:paddingVertical="16dp"
         android:textColor="#ff8b8b8b"
-        android:textSize="12sp"
-        android:padding="18dp" />
+        android:textSize="12sp" />
```

Comment:
We should explicitly specify roboto regular here, right?

Reply:
Nothing was specified in the mock here. But yeah I think it should be that.